cmd/gator/test/gatortest_test.go

```diff
@@ -81,7 +86,8 @@ func Test_formatOutput(t *testing.T) {
             "kind": "kind"
         },
         "violatingObject": {
-            "bar": "xyz"
+            "bar": "xyz",
+            "kind": "kind"
```

Comment:
why the addition of kind here and elsewhere?

Reply:
IIUC, the extra unmarshalling threw an error since we want `kind`.

I think the changes in this file will be reverted per the comment you made here:  https://github.com/open-policy-agent/gatekeeper/pull/2544/files#r1091320800